actions/plugins.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Client4} from 'mattermost-redux/client';
+
+export async function doPluginAction(pluginId, requestURL, body) {
+    await Client4.executePluginIntegration(pluginId, requestURL, body);
```

Comment:
What's the plan for this function? Will we be doing processing like "the cloud app told me to open this modal, or redirect to this URL, so I'll do that now in this function."?

Reply:
Yes, that is exactly the objective.